modules/social_features/social_group/src/Plugin/views/field/SocialGroupViewsBulkOperationsBulkForm.php

```diff
@@ -199,25 +199,31 @@ public function viewsFormValidate(&$form, FormStateInterface $form_state) {
     if ($this->view->id() === 'group_manage_members') {
       $user_input = $form_state->getUserInput();
       $available_options = $this->getBulkOptions();
+
+      $selected_actions = array_combine(
+        array_keys($this->options['selected_actions']),
+        array_column($this->options['selected_actions'], 'action_id')
+      );
+
       // Grab all the actions that are available.
       foreach (Element::children($this->actions) as $action) {
-        // If the option is not in our selected options, next.
-        if (($action_key = array_search($action, array_column($this->options['selected_actions'], 'action_id'))) === FALSE) {
-          continue;
-        }
-
-        /** @var \Drupal\Core\StringTranslation\TranslatableMarkup $label */
-        $label = $available_options[$action_key];
-
-        // Match the Users action from our custom dropdown.
-        // Find the action from the VBO selection.
-        // And set that as the chosen action in the form_state.
-        if (strip_tags($label->render()) === $user_input['op']) {
-          $user_input['action'] = $action_key;
-          $form_state->setUserInput($user_input);
-          $form_state->setValue('action', $action_key);
-          $form_state->setTriggeringElement($this->actions[$action]);
-          break;
+        // Check if we have the command.
+        if (is_array($selected_actions) &&
```

Comment:
I think we can do this more readable:
- Change the `if` structure
- Remove extra blank line right after `if`
```
if (
  is_array($selected_actions) &&
  ($action_key = array_search($action, $selected_actions)) !== FALSE
) {
  /** @var \Drupal\Core\StringTranslation\TranslatableMarkup $label */
  $label = $available_options[$action_key];
```